### ISSUE TITLE
Puzzle Cover Problems (Clock and Square-1)

### ISSUE THREAD

**Issue by sarahstrong314** [thewca/wca-regulations#67]
Since we can no longer use score cards to cover puzzles, it makes it difficult to not affect the state of Clock and Square-1 when they're under a cardboard/paper cover.

Specifically, at Worlds, because of the size of the covers, Square-1s were often on their sides (U/D were on R/L) after the covers were lifted, and it was difficult to take the cover off of the Clocks without tipping them over and affecting the pins. When I was judging for these two events, I was extra careful when placing the puzzles in the covers and lifting the covers, but accidents involving affecting the state of the puzzles still happened.

The issue was that the covers were too large for Square-1, but too small for Clock. It would have been much easier if the Clock covers were flat and tall like a Clock (with extra room so that the clock isn't being scraped by the covers when being lifted), and if the Square-1 covers were shorter so that they wouldn't tumble on their sides (which applies moves to the U/D layers).

I don't think it's realistic to have specific cover size requirements for these events, but what about suggesting what the ideal cover sizes are in the guidelines?

**Comment by iVinshe** [comment]
So you're suggesting different-sized covers for different puzzles as a
Guideline? Could be good. As an organizer, I would opt with our current
boxes because I'd rather not have to carry two sets of boxes to the venue.

**Comment by lgarron** [comment]
I would like to propose a new idea: work with a manufacturer (Meffert's? cube4you?) to create a set of inexpensive puzzle covers / transport containers.

I believe we could solve all or most of the following issues:
- Making sure clocks stand up at the table.
- Holding different kinds of puzzles comfortably.
- Keeping puzzles in the container during transport.
- Keeping the scorecard with the puzzle easily.
- Keeping the scrambling table organization clean.
- Easily stacking puzzle containers to clear the table, or making them compact for transport.
- Possibly indicating the state of the puzzle (new/scrambled/solved/something weird like extra scramble).
- Keeping the puzzle oriented a specific way when the competitor receives it.
- Collecting and stacking individual scorecards after competitors are done with their attempts.

I'm currently imagining this as two parts: a base and a cover. The base would have a special shape that accommodates different puzzles, such as a ridge that can help a clock stand up clock. Every competitor's puzzle would travel with a base during the round, and the scrambler would place the cover over both. The cover wouldn't need to be too fancy, but would ideally at least have something like a clip that a scorecard can be attached to, and possibly a simple indicator switch.

For clock, the competitor could use the base to put the puzzle back onto the table standing. For other puzzles, if competitors mind having a base plate on the stackmat, the judge could remove it right after the beginning of inspection. (Or we could just say the base plate is part of the official speedsolving procedure.)

I think both of these can be designed as plastic moulds that are cheap to make, and easy/compact to stack.

**Comment by sarahstrong314** [comment]
That would be awesome!
I especially like the idea of each cover having base. The runners would no longer have to turn the covers upside-down when they're bringing the puzzles from the scrambling stations to the timing stations. I believe this was the main reason Square-1s were on their sides.

edit:
If the covers are shaped like [this](http://image.shutterstock.com/display_pic_with_logo/61739/61739,1250506491,2/stock-photo-gift-boxes-and-lids-on-white-background-35398027.jpg), where the lid is the base, the scorecards can be tucked between the base and the cover .

**Comment by lgarron** [comment]
One more thing this could help with: giving the puzzle to the competitor with a consistent/random orientation (just mark which side of the base should face the competitor, and possibly make the cover conform to the same orientation).

**Comment by sarahstrong314** [comment]
For the Battle of Waterloo competition a few weeks ago, I made box-shaped covers with bases (lids) out of poster paper to use as puzzle covers. 

They were very nice to have at scrambling stations, once we had a good system for sorting scrambled and unscrambled puzzles, since they were easy to handle and it was easy to keep the scorecards with the right puzzles and the right covers. At the timing stations, people tended not to like them, since some of the judges often forgot to remove the lid from the mat during inspection.

We held square-1 and clock, and as far as I know we didn't have any problems with square-1s being on their side, or clocks falling over (except for a few times when the cover was lifted too quickly). 

There were some judges who found that the lids got in the way too much, and preferred to remove the lids from under the cover before inspection (see http://www.youtube.com/watch?v=GIme6aZWyuk).

Overall, there were some mixed feelings about whether using the bases were helpful or not. They were definitely nice to have at scrambling stations and it would have been even nicer if they were stackable, but several people found them were problematic at the timing stations, mostly because they gave the judges the extra responsibility to make sure the lids weren't in the way.

**Comment by lgarron** [comment]
I haven't replied to this yet, but those bases are quite "sunk". I'm imagining something that's more of a small platform.

Leaving the base: It might seem annoying at first to place the cube back on those, but it certainly forces the competitor to place them in a standard way (none of this "cube-on-timer" nonsense).

Then again, it would be nice to have the mat for puzzle defect/stopping the solve. Then the judge might as well b in charge of removing it early.

**Comment by sarahstrong314** [comment]
I came up with a better idea for the model of the lid after the competition. If one of the four short sides weren't there, then the judge could have just slid it away while keeping the puzzle under the cover. That would have fixed all of the problem and inconveniences we had. The runners would still be able to carry the covers with the lids under them in the same manner, and the scramblers can still tuck the scorecard into the lid, and additionally the judge would be able to easily and quickly remove the lid right before inspection time. 

I didn't think about whether the base should replace the stackmat as the place to put the cube right before the solve starts. It seems inconvenient that the competitor would have that additional responsibility, and unless the base has to be in a specific location every time, it would be inconsistent and unfair.

**Comment by Laura-O** [comment]
There have been several issues concerning the clock covers at the European Championships. They were simple card boxes which were helpful to cover and carry the puzzles, but didn't prevent the pins from being pushed.

So I finally implemented an idea for simple covers I had some time ago.
Basically this is a box in the size of the clock. In contrast to other covers used there is a hole in the top which allows to grip the upper part of the clock. This is so small that it is not possible to see the relevant parts of the puzzle. I attached some photos from a prototype I just made from a sheet of paper. It would probably be useful to close the sides (i.e. make this a box) to prevent the clock from rolling to the side and to use thicker paper.

This cover violates A2c1 ("The scrambler places a cover over the scrambled puzzle that makes it impossible for any competitors or spectators to see any part of the puzzle. The cover remains over the puzzle until the beginning of the attempt.") because it's possible to see the upper part of the clock. However this isn't an issue in this case since the competitor doesn't have any advantages through this.

![clock_cover3](https://cloud.githubusercontent.com/assets/362249/3913190/2c3427be-2333-11e4-964a-fb3add270d67.JPG)
![clock_cover2](https://cloud.githubusercontent.com/assets/362249/3913189/2c308816-2333-11e4-913b-6b65dd9c5f88.JPG)
![clock_cover1](https://cloud.githubusercontent.com/assets/362249/3913191/2c353528-2333-11e4-95f4-2081b724cda1.JPG)
